tools/codeformat.py

```diff
@@ -92,9 +92,11 @@ def fixup_c(filename):
     # Write out file with fixups.
     with open(filename, "w", newline="") as f:
         dedent_stack = []
+        line_number = 0
         while lines:
             # Get next line.
             l = lines.pop(0)
+            line_number += 1
```

Comment:
```suggestion
        for line_number, l in enumerate(lines):
```

And `l_next = lines[0]` changes to `l_next = lines[line_number]`